plugins/modules/net_tools/nmcli.py

```diff
@@ -345,12 +455,141 @@
+                type: str
+            generate-mac-address-mask:
+                description: With "cloned-mac-address" setting "random" or "stable", by default all bits of the MAC address are scrambled and a
+                             locally-administered, unicast MAC address is created. This property allows to specify that certain bits are fixed. Note that the
+                             least significant bit of the first MAC address will always be unset to create a unicast MAC address. If the property is NULL, it
+                             is eligible to be overwritten by a default connection setting. If the value is still NULL or an empty string, the default is to
+                             create a locally-administered, unicast MAC address. If the value contains one MAC address, this address is used as mask. The set
+                             bits of the mask are to be filled with the current MAC address of the device, while the unset bits are subject to randomization.
+                             Setting "FE:FF:FF:00:00:00" means to preserve the OUI of the current MAC address and only randomize the lower 3 bytes using the
+                             "random" or "stable" algorithm. If the value contains one additional MAC address after the mask, this address is used instead of
+                             the current MAC address to fill the bits that shall not be randomized. For example, a value of
+                             "FE:FF:FF:00:00:00 68:F7:28:00:00:00" will set the OUI of the MAC address to 68:F7:28, while the lower bits are randomized. A
+                             value of "02:00:00:00:00:00 00:00:00:00:00:00" will create a fully scrambled globally-administered, burned-in MAC address. If the
+                             value contains more than one additional MAC addresses, one of them is chosen randomly. For example,
+                             "02:00:00:00:00:00 00:00:00:00:00:00 02:00:00:00:00:00" will create a fully scrambled MAC address, randomly locally or globally
+                             administered.
+                type: str
+            hidden:
+                description: If TRUE, indicates that the network is a non-broadcasting network that hides its SSID. This works both in infrastructure and AP
+                             mode. In infrastructure mode, various workarounds are used for a more reliable discovery of hidden networks, such as
+                             probe-scanning the SSID. However, these workarounds expose inherent insecurities with hidden SSID networks, and thus hidden SSID
+                             networks should be used with caution. In AP mode, the created network does not broadcast its SSID. Note that marking the network
+                             as hidden may be a privacy issue for you (in infrastructure mode) or client stations (in AP mode), as the explicit probe-scans are
+                             distinctly recognizable on the air.
```

Comment:
1. Boolean values are `true` and `false`.
2. Break up too long paragraphs into multiple paragraphs.
```suggestion
                description:
                  - If C(true), indicates that the network is a non-broadcasting network that hides its SSID. This works both in infrastructure and AP
                    mode.
                  - In infrastructure mode, various workarounds are used for a more reliable discovery of hidden networks, such as
                    probe-scanning the SSID. However, these workarounds expose inherent insecurities with hidden SSID networks, and thus hidden SSID
                    networks should be used with caution.
                  - In AP mode, the created network does not broadcast its SSID. Note that marking the network
                    as hidden may be a privacy issue for you (in infrastructure mode) or client stations (in AP mode), as the explicit probe-scans are
                    distinctly recognizable on the air.
```